lib/plugins/aws/info/getStackInfo.js

```diff
@@ -57,6 +44,24 @@ module.exports = {
       }
 
       return BbPromise.resolve();
+    })
+    .then(() => this.provider.getAccountId())
+    .then((accountId) => {
+      const stage = this.provider.getStage();
+      const region = this.provider.getRegion();
```

Comment:
Yes. I'll update the code so that `this.options.region` is used. This is used anyway for the retrieval. See here: https://github.com/serverless/serverless/blob/a6c3240d62a72cca306b9b5b391fb60f648b79de/lib/plugins/aws/info/getStackInfo.js#L25-L26